build.sh

```diff
@@ -206,6 +207,7 @@ if completeBuild || hasArg libcuml || hasArg prims || hasArg bench || hasArg pri
           -DBUILD_CUML_MPI_COMMS=${BUILD_CPP_MG_TESTS} \
           -DBUILD_CUML_MG_TESTS=${BUILD_CPP_MG_TESTS} \
           -DBUILD_STATIC_FAISS=${BUILD_STATIC_FAISS} \
+          -DBUILD_GTEST=${BUILD_GTEST} \
```

Comment:
This flag as it is gone in #3844, gtest gets built automatically if its not found. Do you think we should keep the option of forcing a manual build of gtest?

Reply:
Thanks for the clarification.  Left a few comments in the PR you referenced.  Will close this one.